analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -481,12 +470,14 @@ class NPM : PackageManager() {
                 packages.map { it.toCuratedPackage() }.toSortedSet())
     }
 
+    protected open fun getLockFiles() = listOf("npm-shrinkwrap.json", "package-lock.json")
```

Comment:
This should be an `open val`, not a function:
https://kotlinlang.org/docs/reference/classes.html#overriding-properties

Reply:
Done.